packages/gatsby-theme-store/src/components/ShippingSimulator/index.tsx

```diff
@@ -1,59 +0,0 @@
-import React from 'react'
```

Comment:
Why removed this component?

Reply:
It looks like it was only used in Marin Brasil and Gimenes made [some changes there](https://github.com/vtex-sites/marinbrasil.store/pull/439)

Reply:
The idea is to have the glue code between sdk and ui in the store's own code